### ISSUE TITLE
Wither Disguise HitBox (MythicMob)

### ISSUE THREAD

**Issue by snakehugo** [libraryaddict/LibsDisguises#582]
Here is my Mythicmob [mob](https://pastebin.com/h70EsmzY) (the link will expire in 1 month)

I created a Wither_Skeleton disguised in a Wither Boss.
The issue is that the mob doesn't take the hitbox of the Wither Boss but the hitbox of the Wither_Skeleton when you shoot with a bow (it works when you hit it with an sword for example).
Indeed, I already turned the "**ModifyBoundingBox**" to true but it doesn't seem to be working.

Using :
- PaperSpigot **1.16.5** #472
- MythicMobs 4.12.0
- ProtocolLib 4.6.0
- LibsDisguises 10.0.25

**Comment by snakehugo** [comment]
I'm up to date

**Comment by libraryaddict** [comment]
I wasn't able to test myself but others tested for me and reported no issues. You are not using a custom wither size right? Does a rabbit work fine?

**Comment by libraryaddict** [comment]
Scratch that

**Comment by snakehugo** [comment]
In can be more precise:

You can't shoot the mob in the head with a bow as you could if it was a real wither

**Comment by snakehugo** [comment]
I suspect the fact that when you shoot an arrow, it not longer a PvP mode, we need to check the shooter. So that's why the disguise hitbox works fine when it's a punch but no longer when it's an arrow.

**Comment by libraryaddict** [comment]
No, it's not to do with that. Its to do with hitboxes not being modified properly.

However I currently have a lot of other stuff to do and this doesn't rank very high. It'll be fixed before the next release.

**Comment by libraryaddict** [comment]
Disguised a wither skeleton as a wither on Spigot 1.16.5 and no issues occured. This is likely a paper issue.
